src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,282 @@
+1. Locate the VPC ID of the VPC you want to create your endpoint in. Cockroach Labs recommends that you create the VPC in the availability zones where your cluster is deployed, and as near as possible to the availability zones where your application or service is deployed. You can also choose a different VPC as long as it is peered to the VPC your application is running in and the private endpoint is configured to be DNS-accessible across the peered VPCs.
+
+1. On the **Your VPCs** page, locate the IPv4 CIDR corresponding to the VPC you chose in Step 4.
+
+1. Click **Subnets** in the sidebar.
+
+1. Locate the subnet IDs corresponding to the VPC you chose in Step 4.
+
+1. Click **Security Groups** in the sidebar.
+
+1. Click **Create security group** to create a security group within your VPC that allows inbound access from your application or source program on Port 26257: {{ security_group_substeps }}
+
+Use either the Amazon VPC Console or the [AWS Command Line Interface (CLI)](https://aws.amazon.com/cli/) to continue:
+
+<div class="filters clearfix">
+  <button style="width: 15%" class="filter-button" data-scope="aws-console">AWS Console</button>
+  <button style="width: 15%" class="filter-button" data-scope="aws-cli">AWS CLI</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="aws-console">
+
+1. Click **Endpoints** in the sidebar.
+1. Click **Create Endpoint**.
+1. On the **Create Endpoint** page, for the **Service Category** field, select **Find service by name**.
+1. In the **Service Name** field, enter the **Service Name** copied from the connection modal in [Step 1](#step-1).
+1. Click **Verify**.
+1. In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
+1. Verify that the subnets are pre-populated.
+1. In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
+1. Click **Create Endpoint**.
+
+    The VPC Endpoint ID displays.
+
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="aws-cli">
+
+1. Substitute the values from the previous steps and run the following AWS CLI command:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    aws ec2 create-vpc-endpoint --region {REGION} \
+                                --vpc-id {VPC_ID} \
+                                --subnet-ids {SUBNET_ID1} {SUBNET_ID2} \
+                                --vpc-endpoint-type Interface \
+                                --security-group-ids {SECURITY_GROUP_ID1} {SECURITY_GROUP_ID2| \
```

Comment:
```suggestion
                                --security-group-ids {SECURITY_GROUP_ID1} {SECURITY_GROUP_ID2} \
```
Is this causing the breakage?

Reply:
I did some playing around, and I think I found the source of the breakage:

- Single-level `section`s with Markdown render as Markdown.
- Double-level `section`s with Markdown in the inner `section` will render the Markdown in the inner `section`.
- Double-level `section`s with HTML in the outer `section` and Markdown in the inner `section` will render both items as we expect.
- Double-level `section`s with Markdown in both the outer and inner `section`s will only render Markdown in the inner `section`. The outer `section`'s Markdown will be interpreted as raw text in a `p` tag.

The solution here is to end the outer section and then start a new section that consists of only HTML elements **or** `sections` containing markdown with the `markdown="1"` attribute.

I did update this line and also balanced the `section` tags. Gonna send a new PR.

@mdlinville